command/build.py

```diff
@@ -1,8 +1,37 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
 
-    def __init__(self):	
+    def __init__(self):
         pass
+
+    def _get_empty_bundle(self):
+         return dict(
+            data = dict(
+                customResourceDefinitions = [],
+                clusterServiceVersions = [],
+                packages = [],
+            )
+        )
     
-    def build(self):
-        print("Hello, world. The build feature is currently under construction.")
+    def _get_field_entry(self, yamlContent):
+        yaml_type = identify.get_operator_artifact_type(yamlContent)
```

Comment:
Given we are going to be using `operator-registry`, we should be adopting [definitions](https://github.com/operator-framework/operator-registry#manifest-format) that are being used there.

Reply:
@SamiSousa Artifact in this context (identify.py) does not refer to the bundle, it refers to any subsection of yaml that is being created. Since we don't have a term for a generic 'component' I think `Artifact` is fine in this context.